packages/docusaurus/lib/commands/build.js

```diff
@@ -40,7 +41,7 @@ function compile(config) {
 
 module.exports = async function build(siteDir, cliOptions = {}) {
   process.env.NODE_ENV = 'production';
-  console.log('Build command invoked ...');
+  console.log(chalk.blue('Creating an optimized production build...\n'));
```

Comment:
We'll eventually need a logging abstraction to help us print logs with different colors depending on the severity of the log (and also helping to show only those that the user is currently interested in).